test/test_gpu3.cpp

```diff
@@ -7002,7 +7004,7 @@ TEST_F(NVFuserTest, FusionIntegerType_CUDA) {
     fusion.addInput(tv0);
 
     auto i2 = IrBuilder::create<Scalar>(int64_val, DataType::Int);
-    auto i3 = IrBuilder::create<Scalar>(int_val, DataType::Int32);
+    auto i3 = IrBuilder::create<Scalar>((int64_t)int_val, DataType::Int32);
```

Comment:
@zasdfgbnm
With how we initialize `Scalar` instances perhaps this test case lost its purpose?

Reply:
I think this test is fine. This test mostly tests type inference and lowering, and the recent `Scalar` refactor is only related to host evaluation.